src/index.js

```diff
@@ -19,7 +21,7 @@ const schema = require('./options.json');
 
 const defaults = {
   cacheContext: '',
-  cacheDirectory: path.resolve('./node_modules/.cache/.cache-loader'),
+  cacheDirectory: findCacheDir({ name: 'cache-loader' }) || os.tmpdir(),
```

Comment:
no need `os.tmpdir`, `find-cache-dir` already has fallback when directory was not found

Reply:
I think we are good here as reading the documentation of `find-cache-dir` doesn't provide any hint about a fallback. Are u okay with it @evilebottnawi ?

Reply:
@mistic will be great send a PR about this in `find-cache-dir` too, but not related to this PR

Reply:
I'll do it @evilebottnawi  😃